File: src/main/java/com/github/theword/queqiao/tool/payload/MessagePayload.java
package com.github.theword.queqiao.tool.payload;

import com.google.gson.JsonElement;


/**
 * 消息负载类
 */
public class MessagePayload {

    private JsonElement message;

    public MessagePayload(JsonElement message) {
        this.message = message;
    }

    public MessagePayload() {
    }

    public JsonElement getMessage() {
        return message;
    }

    public void setMessage(JsonElement message) {
        this.message = message;
    }
}

File: src/main/java/com/github/theword/queqiao/tool/payload/MessagePayload.kt
package com.github.theword.queqiao.tool.payload

import com.github.theword.queqiao.tool.payload.modle.component.CommonTextComponent
import com.google.gson.annotations.SerializedName
import java.util.stream.Collectors

open class MessagePayload {
    @SerializedName("message")
    var message: List<MessageSegment> = listOf()
    override fun toString(): String {
        return message.stream()
            .map { obj: MessageSegment -> obj.data }
            .map { obj: CommonTextComponent -> obj.text }
            .collect(Collectors.joining())
    }
}
